### PR TITLE
Fix: (java) remove @ from operators highlight

### PR DESCRIPTION
**Pull request by Delice0**
Fixes: https://github.com/nvim-treesitter/nvim-treesitter/issues/5512

### CODE REVIEW

**Comment by amaanq** [comment]
As discussed there, it would be better fit as `@annotation` then, can you do so?

File: queries/java/highlights.scm [hunk 1]
```diff
@@ -35,14 +35,15 @@
 ; Annotations
 
 (annotation
-  name: (identifier) @attribute)
+  "@" @annotation
+  name: (identifier) @annotation.name)
 (marker_annotation
-  name: (identifier) @attribute)
+  "@" @annotation
+  name: (identifier) @annotation.name)
```
Comment (lucario387) [inline queries/java/highlights.scm:42]: All of these should be `@attribute`, the CI doesn't accept annotation as a valid name

Comment (Delice0) [reply queries/java/highlights.scm:42]: I don't understand. @amaanq asked it to be @annotation or am I missing something here?

Comment (amaanq) [reply queries/java/highlights.scm]: whoops, mixed up annotation and attribute, but yeah `@attribute` is fine

Comment (Delice0) [reply queries/java/highlights.scm]: I changed it. Please review again

**Review by amaanq — APPROVED** [review]

**Pull request merged**